extensions/amp-a4a/0.1/secure-frame.js

```diff
@@ -62,13 +63,19 @@ export const createSecureDocSkeleton = (url, sanitizedHeadElements, body) =>
 
 /**
  * Create iframe with predefined CSP and sandbox attributes for security.
- * @param {!Document} document
+ * @param {!Window} win
  * @param {string} title
  * @param {string} height
  * @param {string} width
  * @return {!HTMLIFrameElement}
  */
-export function createSecureFrame(document, title, height, width) {
+export function createSecureFrame(win, title, height, width) {
+  let sandbox = sandboxVals.join(' ');
+  if (Services.platformFor(win).isSafari()) {
+    sandbox += ' allow-scripts';
```

Comment:
might be cleaner doing `sandboxVals.push('allow-scripts')`

Reply:
This was my first instinct, I think that would keep adding to the array every time it is called?

Reply:
Didn't notice it's a constant. How about

```
sandbox = sandboxVals.concat(['allow-scripts']);
```

Reply:
Look like the only usage of `sandboxVal` is here. Why don't we set the whole space-separated sandbox attribute as a string constant?

Reply:
changed to string.